### PR TITLE
lodash: signatures of _.isFinite have been changed

### PR DESCRIPTION
**Pull request by chrootsu**
https://lodash.com/docs#isFinite
https://lodash.com/docs#_ (description of the chainable wrapper)

### CODE REVIEW

**Comment by vvakame** [comment]
thanks mate!

File: lodash/lodash.d.ts [hunk 1]
```diff
@@ -9928,11 +9928,13 @@ declare module _ {
...
          * Checks if value is a finite primitive number.
+         *
          * Note: This method is based on Number.isFinite.
+         *
          * @param value The value to check.
          * @return Returns true if value is a finite number, else false.
-         **/
-        isFinite(value?: any): value is number;
+         */
+        isFinite(value?: any): boolean;
```
Comment (vvakame) [inline lodash/lodash.d.ts]: I'm interested in this change. why do you need this change?
http://goo.gl/dmCZHX

Comment (chrootsu) [inline lodash/lodash.d.ts]: @vvakame There is one case where it is not works http://goo.gl/Weq5aV . Am I wrong?

Comment (vvakame) [inline lodash/lodash.d.ts]: make sence, I think you are right :+1:

**Pull request merged**